src/clients/api/queries/useGetTransactions.ts

```diff
@@ -15,9 +15,9 @@ type Options = QueryObserverOptions<
 
 const useGetTransactions = (params: IGetTransactionsInput, options?: Options) =>
   useQuery([FunctionKey.GET_TRANSACTIONS, params], () => getTransactions(params), {
-    ...options,
```

Comment:
This is inconsistent with other queries that also use pagination and `keepPreviousData`.
Being able to override this option is the right approach in my opinion, since there could be instances where we only one to load one page at a time and not keep the previous data.
Also, allowing options to be passed but not overriding the default ones will create issues for devs who won't understand why the options they pass to the queries aren't taken in consideration.

Reply:
Yeah I commented on the others as well. The comment was meant to explain why. Even loading one page it would make sense to keep the previous data

Reply:
Not in the case I described where you only load one page and display that one page, plus the rest of my comment